src/vos/vos_pool.c

```diff
@@ -151,6 +145,18 @@ pool_hop_free(struct d_ulink *hlink)
 	if (pool->vp_dying)
 		vos_delete_blob(pool->vp_id);
 
+	if (pool->vp_meta_context != NULL) {
+		rc = bio_mc_close(pool->vp_meta_context, bm_flags);
```

Comment:
see above, vos_delete_blob() has been called.

Reply:
yup, should be moved before delete.